Add Note about cluster scaling [CN-1195] (#204)
hazelcast/hazelcast-platform-operator-docs: docs/modules/ROOT/pages/cp-subsystem.adoc

[source,yaml,subs="attributes+"]
----
include::ROOT:example$/hazelcast-cp-subsystem-persistence.yaml[]
----

> NOTE: The cluster cannot be scaled up or down with the CP subsystem enabled. Although it is allowed to pause the cluster, i.e., scaling it to 0, when starting the cluster, you must scale only to the original cluster size; otherwise, it will fail to start.

Reviewer: ```suggestion
NOTE: The cluster cannot be scaled up or down if the CP subsystem is enabled. However, you can pause the cluster by scaling it to 0 when starting the cluster. You must only scale to the original cluster size or it fails to start.
```

Reviewer: Not convinced by my suggestion, I think it needs something else - could you clarify slightly, please, I am reading: _You can pause the cluster by scaling to zero, but if zero wasn't the original cluster size you cannot restart it_, but that doesn't seem logical.

Is it?

Author: Well, not quite like this. What I intended to say, is that it is allowed to scale to zero, but from zero, it is only allowed to scale to the original value.
For example, let's say we have a cluster of size 3, and we scale it to 0. Then, it is allowed to scale it back to 3, but it is not allowed to scale it to 5, as the original value was 3. Hope I haven't explained it in a more complicated way 😄

Reviewer: Not at all more complicated, nicely done - and I've added the example to the note ;-)

Author: Thanks! 👍
